Zero-Gravity-Store-React/src/Components/Navigation/Navigation.css

```diff
@@ -0,0 +1,190 @@
+.logo-img {
+    width: 85%;
+  }
+  a {
+    text-decoration: none;
+    color: azure;
+  }
+  
+  button {
+    cursor: pointer;
+  }
+  
+  /* .line-height {
+    line-height: 1.6rem;
+  } */
+  
+  /* PRODUCT PAGE CSS */
+  
+  .App {
+    font-family: sans-serif;
+    text-align: center;
+  }
+  
+  .logo-img {
+    width: 85%;
+  }
+  a {
+    text-decoration: none;
+    color: var(--text-color);
+  }
+  li {
+    list-style: none;
+    color: var(--text-color);
+  }
+  .big-wrapper {
+    position: relative;
+    padding: 1.7rem 0 2rem;
+    width: 100%;
+    min-height: 100vh;
+    overflow: hidden;
+    background-color: var(--background-color);
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+  }
+  .hero-logo {
+    display: flex;
+    align-items: center;
+    cursor: pointer;
+    margin-left: 3rem;
+  }
+  .logo-mg {
+    width: 42%;
+    margin-right: -2rem;
+  }
+  .bg-color {
+    color: var(--text-color);
+    font-size: 1.4rem;
+    font-family: monospace;
+    font-weight: 600;
+  }
+  
+  header {
+    position: relative;
+    z-index: 70;
+  }
+  header .container {
+    display: flex;
+    justify-content: center;
+    justify-content: space-between;
+    align-items: center;
+  }
+  .ul-margin {
+    margin-right: 60rem;
+  }
+  button.link-no-style {
+    margin-right: -4px;
+  }
+  
+  .search-bar-btn {
```

Comment:
take care of spacing between the classes

Reply:
ohk i will remove it